samples/Config.CustomConfigurationProviders.Web/Extensions/ConfigurationProviderKeyExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Use the exact license header that's used elsewhere in this repo (though admittedly many files are missing it entirely):

https://github.com/aspnet/Entropy/blob/dac7c2f64567956f3a6eafb24b51f59f62b0b3b3/samples/Mvc.Modules/ModuleFramework/IAsyncModuleFilter.cs#L1-L2